go/types/path_test.go

```diff
@@ -62,14 +62,9 @@ func TestPathIndex(t *testing.T) {
 	assert := assert.New(t)
 
 	var v Value
-	resolvesTo := func(exp, val Value, str string) {
-		// Indices resolve to |exp|.
-		assertResolvesTo(assert, exp, v, str)
-		// Keys resolve to themselves.
-		if exp != nil {
-			exp = val
-		}
-		assertResolvesTo(assert, exp, v, str+"@key")
+	resolvesTo := func(expVal, expKey Value, str string) {
```

Comment:
Test cleanup, I preferred the way I wrote them just now, so I changed the existing ones to match.